redis_json/src/commands.rs

```diff
@@ -1361,15 +1367,42 @@ pub fn json_arr_len<M: Manager>(manager: M, ctx: &Context, args: Vec<RedisString
 }
 
 ///
-/// JSON.ARRPOP <key> [path [index]]
+/// JSON.ARRPOP <key>
+///         [FORMAT {STRING|JSON|EXPAND}]
```

Comment:
So here also the path can not be FORMAT .. very ugly api. I would remove it and match the behaviour with `json_mget`..